src/utils.ts

```diff
@@ -1,4 +1,4 @@
-import { basename } from "path";
+import path, { basename, join } from "path";
```

Comment:
```suggestion
import path, { basename, join } from "path";
import { getRuntimeConfig } from "../runtimeConfig.js";
```

Imports getRuntimeConfig() - necessary to find if the client requires URIDecodeComponent (for future proofing since deluge does not require this)